Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -0,0 +1,646 @@
+    if isinstance(data_obj, dict) and len(data_obj) != 0:
+        for k, v in data_obj.items():
+            if isinstance(data_obj[k], dict) or isinstance(data_obj[k], list):
+                convert_empty_to_null(data_obj[k])
+            if not isinstance(v, int) and (v is None or len(v) == 0):
+                data_obj[k] = None
+    elif isinstance(data_obj, list) and len(data_obj) != 0:
+        for index, item in enumerate(data_obj):
+            convert_empty_to_null(item)
+            if not isinstance(item, int) and (item is None or len(item) == 0):
+                data_obj[index] = None
+
+
+def create_domain_context_outputs(domain_result):
+    """
+    Creates all the context data necessary given a domain result
+    Args:
+        domain_result: DomainTools domain data
+
+    Returns: Dict with context data
+
+    """
+    domain = domain_result.get('domain')
+    ip_addresses = ', '.join([x.get('address').get('value') for x in domain_result.get('ip')])
+    create_date = domain_result.get('create_date').get('value')
+    expiration_date = domain_result.get('expiration_date').get('value')
+    domain_name_servers = ', '.join([x.get('host').get('value') for x in domain_result.get('name_server')])
+    name_servers = domain_result.get('name_server')
+    domain_status = domain_result.get('active')
+
+    proximity_risk_score = 0
+    threat_profile_risk_score = 0
+    threat_profile_threats = ''
+    threat_profile_evidence = ''
+
+    overall_risk_score = domain_result.get('domain_risk').get('risk_score')
+    risk_components = domain_result.get('domain_risk').get('components')
+    if len(risk_components):
+        proximity_data = get_threat_component(risk_components, 'proximity')
+        blacklist_data = get_threat_component(risk_components, 'blacklist')
+        if proximity_data:
+            proximity_risk_score = proximity_data.get('risk_score')
+        elif blacklist_data:
+            proximity_risk_score = blacklist_data.get('risk_score')
+        threat_profile_data = get_threat_component(risk_components, 'threat_profile')
+        if threat_profile_data:
+            threat_profile_risk_score = threat_profile_data.get('risk_score')
+            threat_profile_threats = ', '.join(threat_profile_data.get('threats', []))
+            threat_profile_evidence = ', '.join(threat_profile_data.get('evidence', []))
+
+    website_response = domain_result.get('website_response')
+    google_adsense = domain_result.get('adsense').get('value')
+    google_analytics = domain_result.get('google_analytics').get('value')
+    alexa_rank = domain_result.get('alexa')
+    tags = domain_result.get('tags')
+
+    registrant_name = domain_result.get('registrant_name').get('value')
+    registrant_org = domain_result.get('registrant_org').get('value')
+    domain_registrant_contact = {
+        'Country': domain_result.get('registrant_contact').get('country').get('value'),
```

Comment:
no need to get `registrant_contact` so many times, it is better to init it first `registrant_contact = domain_result.get('registrant_contact', {}` and then use it to get the required fields